src/test/java/solution/EnumBoardTest.java

```diff
@@ -0,0 +1,282 @@
+package solution;
+
+import java.awt.Point;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import solution.Solution;
+import solution.BacktrackingSolution;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
static star import of  e.g. assertions is an exception to the rule I described above - here it's completely fine, as the class is restricted to testing and making various assertions and not having to use the `Assertions.???` each time is comfortable.